src/app/components/SearchBox/SearchBox.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+
+import React, { FormEvent, useState } from 'react';
+import { useRouter } from 'next/navigation';
+import styles from './SearchBox.module.scss';
+
+function SearchBox() {
+  const [input, setInput] = useState('');
+  const router = useRouter();
+
+  const handleSearch = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    router.push(`?q=${input}`);
+  };
+
+  return (
+    <form className={styles.searchBox} onSubmit={handleSearch}>
+      <input type="text" placeholder="Search keywords..." value={input} onChange={(e) => setInput(e.target.value)} />
+      <button type="submit" disabled={!input}>
+        <img src="/Search.svg" alt="search" />
```

Comment:
Please try to always use next <Image /> component

Reply:
Okay, thank you. But I don't understand how to use nextImage in Articles component, because it requires adding hosts to next.config, and all hosts are different and there are a lot of them.